src/static/js/modules/gallery.js

```diff
@@ -0,0 +1,42 @@
+function imageGallery() {
+    const ACTIVE_CLASS = "active";
+    const carouselSlide = document.querySelectorAll(".item-slide");
+    const previews = document.querySelectorAll(".gallery-img");
+    const reset = document.querySelector(".gallery-wrapper-miniture img")
+
+    previews.forEach((preview) => {
+        preview.addEventListener("click", function () {
+            let size = carouselSlide[0].clientWidth;
+            let counter = this.id;
+            previews.forEach((preview) => preview.classList.remove(ACTIVE_CLASS));
+            preview.classList.add(ACTIVE_CLASS);
+            carouselSlide.forEach((cl) => {
+                cl.style.transform = "translateX(" + -size * counter + "px)";
```

Comment:
`" + -size * counter + "px` can be converted into a function you pass those values and it returns the expected translateX value, this way you can implement unit testing on that function.

Reply:
Done